go_optouts/server.py

```diff
@@ -1,8 +1,80 @@
-from api import API
-from store.memory import MemoryOptOutBackend
+import sys
 
-site = API(MemoryOptOutBackend())
-resource = site.app.resource  # expose for twistd web
+from twisted.internet import reactor
+from twisted.python import log
+from twisted.web import http
+from twisted.web.resource import Resource
 
-if __name__ == "__main__":
-    site.app.run('localhost', 8080)
+from confmodel import Config
+from confmodel.fields import ConfigText, ConfigDict
+
+from vumi.utils import build_web_site
+
+import yaml
+
+from .api import API
+from .store.memory import MemoryOptOutBackend
+from .store.riak import RiakOptOutBackend
+
+
+class HealthResource(Resource):
+    isLeaf = True
+
+    def render_GET(self, request):
+        request.setResponseCode(http.OK)
+        request.do_not_log = True
+        return 'OK'
+
+
+def read_yaml_config(config_file):
+    """Parse an (usually) optional YAML config file."""
```

Comment:
I was a bit loathe to make memory the default because it's only really for
testing. Maybe I should fix the docstring rather?